contracts/staking/RewardDelegators.sol

```diff
@@ -103,6 +107,8 @@ contract RewardDelegators is
             tokenList.push(_tokenIds[i]);
             emit AddReward(_tokenIds[i], _rewardFactors[i]);
         }
+
+        _updateThresholdForSelection(500_000); //0.5 million pond
```

Comment:
Initialise function can't be called again because it was already done once, so this line might not be necessary though we can keep it for record in case we want to redeploy at some point

Reply:
If that is the idea then set epochSelector address as well